edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/pvc/KubernetesPvcMapper.cs

```diff
@@ -41,7 +41,7 @@ bool ShouldCreatePvc(Mount mount)
             return this.storageClassName.HasValue || this.persistentVolumeName.HasValue;
         }
 
-        V1PersistentVolumeClaim ExtractPvc(Mount mount, IDictionary<string, string> labels)
+        V1PersistentVolumeClaim ExtractPvc(Mount mount, IDictionary<string, string> labels, KubernetesModule module)
```

Comment:
Which should be the preferred order "mount, label, module," or "module, mount, label?"(@CindyXing 's PR has it in the latter order)

Reply:
Thanks. I'll handle on merge after that PR goes in.